hack/cmd_util.sh

```diff
@@ -193,11 +194,8 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	local end_time=$(os::cmd::internal::seconds_since_epoch)
 	local time_elapsed=$(echo "scale=3; ${end_time} - ${start_time}" | bc | xargs printf '%5.3f') # in decimal seconds, we need leading zeroes for parsing later
 
-	# some commands are multi-line, so we may need to clear more than just the previous line
-	local cmd_length=$(echo "${cmd}" | wc -l)
-	for (( i=0; i<${cmd_length}; i++ )); do
-		os::text::clear_last_line
-	done
+    # clear the preamble so we can print out the success or error message
+	os::text::clear_string "${preamble}"
```

Comment:
It doesn't look like we're really doing anything between the `echo` statements and the `os::text::clear_string` statements...?